MainModule/Server/Commands/Players.lua

```diff
@@ -1150,6 +1150,7 @@ return function(Vargs, env)
 			Args = {"id"};
 			Description = "Prompts yourself to buy the asset belonging to the ID supplied";
 			AdminLevel = "Players";
+			Disabled = Settings.DisableBuyItem ~= false;
```

Comment:
Does this actually do anything though

Reply:
The only plave I have ever seen this used is in settings.Commands

Reply:
Alright so I would merge this but you haven't updated the SettingsDescriptions along with changing the settings

Reply:
yes
https://github.com/Epix-Incorporated/Adonis/blob/1acb8016c55ac2a8bf14983b5cf0baa41a52c09c/MainModule/Server/Core/Admin.lua#L1460-L1462

Reply:
Huh. I guess it does then

Reply:
<img width="474" alt="image" src="https://github.com/Epix-Incorporated/Adonis/assets/64012878/c68d7e14-099d-4c98-a99e-9e8b80499859">
So it works, but it doesn't show up on the ingame settings menu